proxy/stats_handler.go

```diff
@@ -0,0 +1,215 @@
+package proxy
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/gorilla/websocket"
+
+	"github.com/rancherio/websocket-proxy/common"
+)
+
+type StatsHandler struct {
+	backend         backendProxy
+	parsedPublicKey interface{}
+}
+
+func (h *StatsHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
+	multiHost := false
+
+	if strings.HasSuffix(req.URL.Path, "project/") || strings.HasSuffix(req.URL.Path, "service/") {
```

Comment:
@wlan0 Can you change this so that it does not depend on a trailing slash being in the url?